src/Nox.Types.EntityFramework/Types/AutoNumber/AutonumberConverter.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace Nox.Types.EntityFramework.Types;
+
+public class AutoNumberConverter : ValueConverter<AutoNumber, uint>
+{
+    public AutoNumberConverter() : base(autonumber => autonumber.Value, currentNumber => AutoNumber.From(currentNumber)) { }
```

Comment:
The convention is FromDatabase... that skips validation

Reply:
Updated to use `FromDatabase`.